alcs-api/src/board/board.controller.spec.ts

```diff
@@ -116,7 +122,7 @@ describe('BoardController', () => {
     expect(planningReviewService.mapToDtos).toHaveBeenCalledTimes(1);
   });
 
-  it('should call through to amendment service for ceo board', async () => {
+  it('should call through to covenant service', async () => {
```

Comment:
Looks like you renamed the existing test instead of making a new test?

Reply:
let me check, it should be a new one

Reply:
Got it, covenant test is redundant and I deleted it. This one looks like a result of rebase, I'll revet it